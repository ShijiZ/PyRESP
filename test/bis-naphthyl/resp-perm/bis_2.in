test for bis-naphthyl
 &cntrl 
 nmol = 2,
 iqopt = 2,
 ihfree = 1,
 qwt = 0.001,
 ioutopt = 1,
 ipol = 5,
 igdm = 1,
 exc12 = 0,
 exc13 = 0,
 ipermdip = 1,
 pwt = 0.001,
 virtual = 0 
 &end 
    1.0
naphthyl
   -1   29  60
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    6    0  0  0  0  40
    6   -1 -1 -1 -1 -1
    6   -1 -1 -1 -1 -1
    6   -1 -1 -1 -1 
    8   -1 -1
    8   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1    0  0
    1   28 59

    1.0
naphthyl
   -1   29  60
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    6   -1 -1 -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    6    0  0  0  0  40
    6   -1 -1 -1 -1 -1
    6   -1 -1 -1 -1 -1
    6   -1 -1 -1 -1 
    8   -1 -1
    8   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1
    1    0  0
    1   28 59


    2
    1   18    2   18
    2
    1   28    2   28

    2
    1   38    2   38
    2
    1   39    2   39
    2
    1   40    2   40
    2
    1   59    2   59




