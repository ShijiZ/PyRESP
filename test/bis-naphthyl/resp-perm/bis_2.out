
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 test for bis-naphthyl
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 2   iqopt       = 2
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00100000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0
 ipermdip    = 1   pwt         = 0.00100000
 virtual     = 0

 --------------------
 Molecule Information
 --------------------
 Multiple-molecule run of 2 molecules

 Molecule 1 weight:     1.000
 Molecule 1 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
 Number of permanent dipoles: 60
    1    6   -1   -1   -1   -1
    2    6   -1   -1   -1   -1
    3    6   -1   -1   -1   -1
    4    6   -1   -1   -1   -1
    5    6   -1   -1   -1   -1
    6    6   -1   -1   -1   -1
    7    6   -1   -1   -1   -1
    8    6   -1   -1   -1   -1
    9    6   -1   -1   -1   -1
   10    6   -1   -1   -1   -1
   11    1   -1   -1
   12    1   -1   -1
   13    1   -1   -1
   14    1   -1   -1
   15    1   -1   -1
   16    1   -1   -1
   17    1   -1   -1
   18    6    0    0    0    0   40
   19    6   -1   -1   -1   -1   -1
   20    6   -1   -1   -1   -1   -1
   21    6   -1   -1   -1   -1
   22    8   -1   -1
   23    8   -1   -1
   24    1   -1   -1
   25    1   -1   -1
   26    1   -1   -1
   27    1   -1   -1
   28    1    0    0
   29    1   28   59

 Molecule 2 weight:     1.000
 Molecule 2 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
 Number of permanent dipoles: 60
   30    6   -1   -1   -1   -1
   31    6   -1   -1   -1   -1
   32    6   -1   -1   -1   -1
   33    6   -1   -1   -1   -1
   34    6   -1   -1   -1   -1
   35    6   -1   -1   -1   -1
   36    6   -1   -1   -1   -1
   37    6   -1   -1   -1   -1
   38    6   -1   -1   -1   -1
   39    6   -1   -1   -1   -1
   40    1   -1   -1
   41    1   -1   -1
   42    1   -1   -1
   43    1   -1   -1
   44    1   -1   -1
   45    1   -1   -1
   46    1   -1   -1
   47    6    0    0    0    0   40
   48    6   -1   -1   -1   -1   -1
   49    6   -1   -1   -1   -1   -1
   50    6   -1   -1   -1   -1
   51    8   -1   -1
   52    8   -1   -1
   53    1   -1   -1
   54    1   -1   -1
   55    1   -1   -1
   56    1   -1   -1
   57    1    0    0
   58    1   28   59

 -------------------------------
 Charges Equivalence Information 
 -------------------------------
    1   18    2   18
    1   28    2   28

 -----------------------------------------
 Permanent Dipoles Equivalence Information
 -----------------------------------------
    1   38    2   38
    1   39    2   39
    1   40    2   40
    1   59    2   59

 -----------------------------
 Atom/Dipole Ivary Information
 -----------------------------
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    6    0    0    0    0   40
    6   -1   -1   -1   -1   -1
    6   -1   -1   -1   -1   -1
    6   -1   -1   -1   -1
    8   -1   -1
    8   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1    0    0
    1   28   59

    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    6   -1   -1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    6   18   38   39   40   40
    6   -1   -1   -1   -1   -1
    6   -1   -1   -1   -1   -1
    6   -1   -1   -1   -1
    8   -1   -1
    8   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1
    1   28   59
    1   28   59

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =   58
 Total number of permanent dipoles      =  120
 Weight factor on initial charge restraints=  0.001000
 Weight factor on initial dipole restraints=  0.001000
 There are  2 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 Reading esp's for molecule   2
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 ------------------------
 Optimization Information
 ------------------------
 Since iqopt>1,   58 new q0 values and  120 new p0 values will be
 read from file bis_1.chg

 Number of unique UNfrozen charge centers =     2
 Number of unique UNfrozen permanent dipoles =     4

 Non-linear optimization requested.
  qchnge =    2.4662002237E-02
  qchnge =    2.7775192555E-04
  qchnge =    2.0512767875E-06
  qchnge =    1.5372529462E-08

 Convergence in    3 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: naphthyl
 molecule 2: naphthyl

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         6       -0.029084     -0.029084     -1       0.009602
    2         6       -0.002048     -0.002048     -1       0.009998
    3         6       -0.053028     -0.053028     -1       0.008835
    4         6       -0.118641     -0.118641     -1       0.006445
    5         6       -0.133366     -0.133366     -1       0.005999
    6         6       -0.067664     -0.067664     -1       0.008282
    7         6        0.001516      0.001516     -1       0.009999
    8         6       -0.045949     -0.045949     -1       0.009087
    9         6       -0.063998     -0.063998     -1       0.008423
   10         6       -0.030421     -0.030421     -1       0.009567
   11         1        0.118413      0.118413     -1       0.000000
   12         1        0.128482      0.128482     -1       0.000000
   13         1        0.059191      0.059191     -1       0.000000
   14         1        0.060420      0.060420     -1       0.000000
   15         1        0.104603      0.104603     -1       0.000000
   16         1        0.066120      0.066120     -1       0.000000
   17         1        0.009812      0.009812     -1       0.000000
   18         6       -0.011758     -0.007222      0       0.009974
   19         6        0.010286      0.010286     -1       0.009948
   20         6       -0.016195     -0.016195     -1       0.009871
   21         6        0.145558      0.145558     -1       0.005663
   22         8       -0.666554     -0.666554     -1       0.001484
   23         8       -0.666554     -0.666554     -1       0.001484
   24         1        0.049401      0.049401     -1       0.000000
   25         1       -0.016027     -0.016027     -1       0.000000
   26         1        0.016093      0.016093     -1       0.000000
   27         1        0.080061      0.080061     -1       0.000000
   28         1       -0.000686      0.033397      0       0.000000
   29         1        0.072015      0.033397     28       0.000000

   30         6       -0.029084     -0.029084     -1       0.009602
   31         6       -0.002048     -0.002048     -1       0.009998
   32         6       -0.053028     -0.053028     -1       0.008835
   33         6       -0.118641     -0.118641     -1       0.006445
   34         6       -0.133366     -0.133366     -1       0.005999
   35         6       -0.067664     -0.067664     -1       0.008282
   36         6        0.001516      0.001516     -1       0.009999
   37         6       -0.045949     -0.045949     -1       0.009087
   38         6       -0.063998     -0.063998     -1       0.008423
   39         6       -0.030421     -0.030421     -1       0.009567
   40         1        0.118413      0.118413     -1       0.000000
   41         1        0.128482      0.128482     -1       0.000000
   42         1        0.059191      0.059191     -1       0.000000
   43         1        0.060420      0.060420     -1       0.000000
   44         1        0.104603      0.104603     -1       0.000000
   45         1        0.066120      0.066120     -1       0.000000
   46         1        0.009812      0.009812     -1       0.000000
   47         6       -0.011758     -0.007222     18       0.009974
   48         6        0.010286      0.010286     -1       0.009948
   49         6       -0.016195     -0.016195     -1       0.009871
   50         6        0.145558      0.145558     -1       0.005663
   51         8       -0.666554     -0.666554     -1       0.001484
   52         8       -0.666554     -0.666554     -1       0.001484
   53         1        0.049401      0.049401     -1       0.000000
   54         1       -0.016027     -0.016027     -1       0.000000
   55         1        0.016093      0.016093     -1       0.000000
   56         1        0.080061      0.080061     -1       0.000000
   57         1       -0.000686      0.033397     28       0.000000
   58         1        0.072015      0.033397     28       0.000000

 Sum over the calculated charges:     -2.000

       Permanent Dipoles Before & After Optimization
 dip.no   element.no   p(init)        p(opt)     ivary    d(rstr)/dp
    1         6        0.039307      0.039307     -1       0.009307
    2         6       -0.013051     -0.013051     -1       0.009916
    3         6       -0.023297     -0.023297     -1       0.009739
    4         6       -0.036975     -0.036975     -1       0.009379
    5         6       -0.006247     -0.006247     -1       0.009981
    6         6        0.046252      0.046252     -1       0.009076
    7         6        0.012321      0.012321     -1       0.009925
    8         6       -0.011353     -0.011353     -1       0.009936
    9         6       -0.001485     -0.001485     -1       0.009999
   10         6       -0.025163     -0.025163     -1       0.009698
   11         6       -0.028757     -0.028757     -1       0.009611
   12         6        0.060285      0.060285     -1       0.008564
   13         6       -0.029875     -0.029875     -1       0.009582
   14         6       -0.029914     -0.029914     -1       0.009581
   15         6        0.069109      0.069109     -1       0.008227
   16         6       -0.027215     -0.027215     -1       0.009649
   17         6        0.012463      0.012463     -1       0.009923
   18         6        0.014232      0.014232     -1       0.009900
   19         6        0.027800      0.027800     -1       0.009635
   20         6       -0.000049     -0.000049     -1       0.010000
   21         6       -0.026744     -0.026744     -1       0.009660
   22         6        0.008596      0.008596     -1       0.009963
   23         6       -0.030995     -0.030995     -1       0.009552
   24         6        0.021929      0.021929     -1       0.009768
   25         6       -0.032052     -0.032052     -1       0.009523
   26         6       -0.010040     -0.010040     -1       0.009950
   27         6        0.045995      0.045995     -1       0.009085
   28         6       -0.070048     -0.070048     -1       0.008190
   29         6       -0.001501     -0.001501     -1       0.009999
   30         6        0.073953      0.073953     -1       0.008040
   31         1       -0.016643     -0.016643     -1       0.000000
   32         1       -0.091857     -0.091857     -1       0.000000
   33         1       -0.137475     -0.137475     -1       0.000000
   34         1       -0.141183     -0.141183     -1       0.000000
   35         1       -0.086064     -0.086064     -1       0.000000
   36         1       -0.136647     -0.136647     -1       0.000000
   37         1       -0.180286     -0.180286     -1       0.000000
   38         6        0.037564      0.035332      0       0.009429
   39         6        0.011906      0.014555      0       0.009896
   40         6       -0.053748     -0.028699      0       0.009612
   41         6       -0.006297     -0.028699     40       0.009612
   42         6        0.004784      0.004784     -1       0.009989
   43         6        0.048397      0.048397     -1       0.009001
   44         6       -0.062939     -0.062939     -1       0.008463
   45         6        0.006104      0.006104     -1       0.009981
   46         6        0.026650      0.026650     -1       0.009663
   47         6       -0.008856     -0.008856     -1       0.009961
   48         6       -0.006606     -0.006606     -1       0.009978
   49         6       -0.012085     -0.012085     -1       0.009928
   50         6        0.153769      0.153769     -1       0.005452
   51         6       -0.038328     -0.038328     -1       0.009338
   52         6       -0.038328     -0.038328     -1       0.009338
   53         8        0.111278      0.111278     -1       0.006684
   54         8        0.111278      0.111278     -1       0.006684
   55         1        0.030482      0.030482     -1       0.000000
   56         1       -0.072135     -0.072135     -1       0.000000
   57         1       -0.001827     -0.001827     -1       0.000000
   58         1        0.009147      0.009147     -1       0.000000
   59         1       -0.076412     -0.027071      0       0.000000
   60         1        0.021399     -0.027071     59       0.000000

   61         6        0.039307      0.039307     -1       0.009307
   62         6       -0.013051     -0.013051     -1       0.009916
   63         6       -0.023297     -0.023297     -1       0.009739
   64         6       -0.036975     -0.036975     -1       0.009379
   65         6       -0.006247     -0.006247     -1       0.009981
   66         6        0.046252      0.046252     -1       0.009076
   67         6        0.012321      0.012321     -1       0.009925
   68         6       -0.011353     -0.011353     -1       0.009936
   69         6       -0.001485     -0.001485     -1       0.009999
   70         6       -0.025163     -0.025163     -1       0.009698
   71         6       -0.028757     -0.028757     -1       0.009611
   72         6        0.060285      0.060285     -1       0.008564
   73         6       -0.029875     -0.029875     -1       0.009582
   74         6       -0.029914     -0.029914     -1       0.009581
   75         6        0.069109      0.069109     -1       0.008227
   76         6       -0.027215     -0.027215     -1       0.009649
   77         6        0.012463      0.012463     -1       0.009923
   78         6        0.014232      0.014232     -1       0.009900
   79         6        0.027800      0.027800     -1       0.009635
   80         6       -0.000049     -0.000049     -1       0.010000
   81         6       -0.026744     -0.026744     -1       0.009660
   82         6        0.008596      0.008596     -1       0.009963
   83         6       -0.030995     -0.030995     -1       0.009552
   84         6        0.021929      0.021929     -1       0.009768
   85         6       -0.032052     -0.032052     -1       0.009523
   86         6       -0.010040     -0.010040     -1       0.009950
   87         6        0.045995      0.045995     -1       0.009085
   88         6       -0.070048     -0.070048     -1       0.008190
   89         6       -0.001501     -0.001501     -1       0.009999
   90         6        0.073953      0.073953     -1       0.008040
   91         1       -0.016643     -0.016643     -1       0.000000
   92         1       -0.091857     -0.091857     -1       0.000000
   93         1       -0.137475     -0.137475     -1       0.000000
   94         1       -0.141183     -0.141183     -1       0.000000
   95         1       -0.086064     -0.086064     -1       0.000000
   96         1       -0.136647     -0.136647     -1       0.000000
   97         1       -0.180286     -0.180286     -1       0.000000
   98         6        0.037564      0.035332     38       0.009429
   99         6        0.011906      0.014555     39       0.009896
  100         6       -0.053748     -0.028699     40       0.009612
  101         6       -0.006297     -0.028699     40       0.009612
  102         6        0.004784      0.004784     -1       0.009989
  103         6        0.048397      0.048397     -1       0.009001
  104         6       -0.062939     -0.062939     -1       0.008463
  105         6        0.006104      0.006104     -1       0.009981
  106         6        0.026650      0.026650     -1       0.009663
  107         6       -0.008856     -0.008856     -1       0.009961
  108         6       -0.006606     -0.006606     -1       0.009978
  109         6       -0.012085     -0.012085     -1       0.009928
  110         6        0.153769      0.153769     -1       0.005452
  111         6       -0.038328     -0.038328     -1       0.009338
  112         6       -0.038328     -0.038328     -1       0.009338
  113         8        0.111278      0.111278     -1       0.006684
  114         8        0.111278      0.111278     -1       0.006684
  115         1        0.030482      0.030482     -1       0.000000
  116         1       -0.072135     -0.072135     -1       0.000000
  117         1       -0.001827     -0.001827     -1       0.000000
  118         1        0.009147      0.009147     -1       0.000000
  119         1       -0.076412     -0.027071     59       0.000000
  120         1        0.021399     -0.027071     59       0.000000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          3.2877896E+01
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   8.0495202E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.7943824E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         1.5647069E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1       12.92106  -10.94993    6.19934   -2.93616
 #MOL         D          Dx         Dy         Dz
   2       12.92106  -10.94993    6.19934   -2.93616

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X  -20.98826
        Y   17.68079   11.15858
        Z  -11.60127    4.49427    9.82968
 #MOL          X          Y          Z
   2    X  -20.98826
        Y   17.68079   11.15858
        Z  -11.60127    4.49427    9.82968

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   19.03693
        Y    0.00000   13.42530
        Z    0.00000    0.00000  -32.46224
 #MOL          X          Y          Z
   2    X   19.03693
        Y    0.00000   13.42530
        Z    0.00000    0.00000  -32.46224